src/components/ProfileMatch/index.tsx

```diff
@@ -0,0 +1,159 @@
+      : 'No information available about Time Commitment',
+};
+
+const startDateMatch: MatchField<
+  CaseListing | LimitedCaseAssignment | DocumentTranslation
+> = {
+  getMatch: (data, profileData) => {
+    const date =
+      data.listing_type === 'CASE' ? data.upcoming_date : data.deadline;
+    return date
+      ? timestampStringToDate(profileData.start_date) <=
+          timestampStringToDate(date)
+      : undefined;
+  },
+  getText: (data, profileData, match) =>
+    match === undefined
+      ? `No information available about the ${
+          data.listing_type === 'CASE'
+            ? 'Next Court Hearing/Filing Date'
+            : 'Deadline'
+        }`
+      : `The ${
+          data.listing_type === 'CASE'
+            ? 'next court hearing/filing date'
+            : 'deadline'
+        } (${formatTimestamp(
+          data.listing_type === 'CASE' ? data.upcoming_date : data.deadline,
+        )}) is ${
+          match ? 'after' : 'before'
+        } your first available start date (${formatTimestamp(
+          profileData.start_date,
+        )}).`,
+};
+
+const matchIcon = (match: boolean | undefined) => {
+  if (match === undefined) {
+    return <Icon type="gray_dot" />;
+  }
+  if (match) {
+    return <Icon type="green_check" />; // rename to greenCheck later
+  }
+  return <Icon type="redCross" />;
+};
+
+export default function ProfileMatch({
+  listingData,
+  interpretation = false,
+}: {
+  listingData: Listing;
+  interpretation?: boolean;
+}) {
+  const profile = useProfile();
+  if (!profile) {
+    throw new Error(
+      'Profile must be complete before ProfileMatch is displayed.',
+    );
+  }
+  const { languages, profileData } = profile;
+  const matchedLanguages = useMemo(
+    () =>
+      languages
+        .filter(l =>
+          listingData.listing_type === 'DOC' ? l.can_read : l.can_speak,
+        )
+        .map(l => l.language_name)
+        .filter(l => listingData.languages.includes(l)),
+    [languages, listingData.languages, listingData.listing_type],
+  );
+
+  const renderIconGroup = (fields: MatchField<Listing>) => {
+    if (!profileData) {
+      return null;
+    }
+    const match = fields.getMatch(listingData, profileData);
+    return (
+      <Flex $align="center" $gap="16px">
+        <IconDiv>{matchIcon(match)}</IconDiv>
+        <P>{fields.getText(listingData, profileData, match)}</P>
+      </Flex>
+    );
+  };
+
+  return (
+    <Container>
+      <H3>Profile Match</H3>
+      {listingData.listing_type === 'CASE' &&
+        renderIconGroup(timeCommitmentMatch as MatchField<Listing>)}
+      {listingData.listing_type !== 'INT' &&
+        renderIconGroup(startDateMatch as MatchField<Listing>)}
+      <Flex $align="center" $gap="16px">
+        <IconDiv>
+          {listingData.listing_type === 'CASE' &&
+          !interpretation &&
+          matchedLanguages.length === 0 ? (
+            <Icon type="yellowExclamation" />
```

Comment:
Could we also display an exclamation mark for LCA when the languages don't match? I don't think it is expected for legal fellows to know the language of the country they are researching.

Reply:
yes! done!